client/flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,43 +5,28 @@
          In most cases you can leave this as-is, but you if you want to provide
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
-  <application android:name="io.flutter.app.FlutterApplication" android:label="COVID-19" android:icon="@mipmap/ic_launcher" android:allowBackup="false">
```

Comment:
Why remove the name, label, icon and allowBackup attributes?

Reply:
I was wondering the same. Looking at the doc, it only mentions removing the `android:name="io.flutter.app.FlutterApplication"`

Reply:
IIRC, the OWASP review included not preserving the client id (or any other random identifiers) across one user's multiple installs or devices.  Some things to test with the change are:
- if you uninstall and reinstall no client id or push or analytics or other identifiers are preserved across installs
- if you install, then sync the app to your google account, and then restore your apps on another device, no client id or push or analytics or other identifiers are preserved across devices

these settings would need to be tested with all account and device backup/sync settings *enabled* to ensure the app is doing the right thing to prevent their backup/sync.

Reply:
Ah -- The docs mentioned removing the flutter tag from the application -- will add back other tags now

Reply:
@Mohit020888 - see Advay's earlier comment. This is something we should add to the test plan once we enter the final QA process.

Reply:
Thanks Bruno - I've added it to the running privacy QA list here if Mohit wants to take a look at that: https://github.com/WorldHealthOrganization/app/issues/1509

Reply:
@brunobowden - will add some test cases around Advay's suggestion for the final QA process.

Reply:
@advayDev1 - thanks Advay ! I am having a look at it now.